llvm/docs/GitHub.rst

```diff
@@ -219,7 +218,7 @@ commonly used first:
 Branches
 ========
 
-It is possible to create branches that start with `users/<username>/`, however this is
+It is possible to create branches in `llvm/llvm-project/` that start with `users/<username>/`, however this is
```

Comment:
[nit] Wrap at 80 cols?

Not sure whether that's the official format for these files. Suggesting for consistency with the existing stye.

Reply:
I did at first but then I thought well, that's extra diff, but I'll put it back.

Reply:
Done.